src/services/utilities.ts

```diff
@@ -2930,16 +2930,16 @@ function findLinkNameEnd(text: string) {
     return 0;
 }
 
-const carriageReturnLineFeed = "\r\n";
+const lineFeed = "\n";
 /**
- * The default is CRLF.
+ * The default is LF.
  *
  * @internal
  */
 export function getNewLineOrDefaultFromHost(host: FormattingHost, formatSettings?: FormatCodeSettings) {
     return formatSettings?.newLineCharacter ||

```

Comment:
Given host.getNewLine for compilerHost changed to use default as LF (instead of OS dependent) so all the completions will be OS independent right. Just making sure if thats desired or not as i was confused about the desired behaviour from previous discussion..

Reply:
Right, I think that's the general intent - we don't want to be providing any OS-dependent behavior ourselves. It should all come from the editor config.